tools/scripts/update-docker-tags.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash -ex
```

Comment:
It's typically preferable to use `set -ex` instead of putting it on the bangpath, but that's not a blocker.